server/world/world.go

```diff
@@ -665,6 +675,78 @@ func (w *World) enableTimeCycle(v bool) {
 	w.set.TimeCycle = v
 }
 
+// ToggleWeatherCycle enables or disables weather of the World.
+func (w *World) ToggleWeatherCycle(v bool) {
```

Comment:
Would split this into a separate StartWeatherCycle and StopWeatherCycle. (Or whatever naming to be analogue with the time cycle)